README.rst

```diff
@@ -88,37 +88,34 @@ Container Images
 ----------------
 
 Bandit is available as a container image, built within the bandit repository
-using GitHub Actions. The image is available on gchr.io:
+using GitHub Actions. The image is available on ghcr.io:
 
 ```bash
-docker pull gcr.io/pycqa/bandit/bandit:py312-arm64
+docker pull ghcr.io/pycqa/bandit/bandit
 ```
 
 The image is built for the following architectures:
 
 * amd64
 * arm64
+* armv7
+* armv8
 
-The image is tagged with the Python version and architecture, for example:
+To pull a specific architecture, use the following format:
 
-* py312-amd64
-* py312-arm64
-
-Python versions supported are:
+```bash
+docker pull ghcr.io/pycqa/bandit/bandit:<tag>-<arch>
```

Comment:
Is this still accurate? I don't think we are publishing image tags like this any longer, unless the `build-push-images` action is doing some magic?

Reply:
No, that's wrong I replaced that. Good catch / my bad.